deployment/helm/synopsys-alert/templates/alert.yaml

```diff
@@ -61,6 +61,10 @@ spec:
             protocol: TCP
         resources:
           {{- toYaml .Values.alert.resources | nindent 12 }}
+        {{- with .Values.alert.securityContext }}
+        securityContext:
+{{ toYaml . | indent 10 }}
```

Comment:
in cfssl, it's 12 and here it's 10.. can you please fix it

Reply:
That's actually the correct indentation. Making the cfssl indent 10 results in:
```
securityContext:
runAsGroup: 2
volumeMounts:
  - mountPath: /etc/cfssl
  name: dir-cfssl
```

Reply:
nope..  runAsGroup should go inside securityContext